pyproject.toml

```diff
@@ -0,0 +1,142 @@
+[build-system]
+requires = ["setuptools>=61.0", "wheel"]
+build-backend = "setuptools.build_meta"
+
+[project]
+name = "wiki2beamer"
+version = "0.10.0"
+description = "Create LaTeX-beamer presentations with a wiki syntax"
+readme = "README.md"
+license = {text = "GPL-2.0-or-later"}
+requires-python = ">=3.7"
+authors = [
+    {name = "Michael Rentzsch", email = "michael.rentzsch@repc.de"},
+    {name = "Kai Dietrich", email = "mail@cleeus.de"},
+]
+maintainers = [
+    {name = "Valentin Haenel", email = "valentin@haenel.co"},
+]
+keywords = ["latex", "beamer", "presentation", "wiki"]
+dependencies = []
+classifiers = [
+    "Development Status :: 5 - Production/Stable",
+    "Environment :: Console",
+    "Intended Audience :: Science/Research",
+    "License :: OSI Approved :: GNU General Public License v2 or later (GPLv2+)",
+    "Programming Language :: Python :: 3",
+    "Programming Language :: Python :: 3.7",
+    "Programming Language :: Python :: 3.8",
+    "Programming Language :: Python :: 3.9",
+    "Programming Language :: Python :: 3.10",
+    "Programming Language :: Python :: 3.11",
+    "Topic :: Text Processing :: Markup",
+    "Topic :: Utilities",
+]
+
+[project.optional-dependencies]
+dev = [
+    "pytest>=7.4.0",
+    "pytest-cov>=6.0.0",
+    "ruff>=0.1.0",
```

Comment:
are ruff and mypy not managed by pre-commit and doesn't that mean they end up in their own virtual env type thing?

Reply:
They are.  It is more personal preference to have them available in the project .venv so that I can just run them without running entire pre-commit.   Also,   I believe vscode needs them to be findable.

Reply:
yeah, ok, why not they are dev deps, so that should be ok.